tools/wheelmaker.py

```diff
@@ -452,25 +463,25 @@ def parse_args() -> argparse.Namespace:
     return parser.parse_args(sys.argv[1:])
 
 
+def _parse_file_pairs(content: List[str]) -> List[List[str]]:
+    """
+    Parse ; delimited lists of files into a 2D list.
+    """
+    return [i.split(";") for i in content or []]
```

Comment:
Am I correct to assume that we will always split `i` into 2 elements? Maybe we should pass the following:
```suggestion
    return [i.split(";", 1) for i in content or []]
```